python/lsst/eotest/image_utils.py

```diff
@@ -289,6 +449,10 @@ def unbias_and_trim(im, overscan, imaging=None, dxmin=5, dxmax=2, bias_method='r
     """
 
     im -= bias_image(im, overscan, dxmin=dxmin, dxmax=dxmax, bias_method=bias_method, **kwargs)
+    bias_method_col = kwargs.get('bias_method_col', 'None')
```

Comment:
I'd make the default `None` and below would do `if bias_method_col is not None:`.

Reply:
This is the same issue that the input might be coming from a yaml file, or a command line argument and might be used to make the output file name so it is easier to keep it as a string.   I could do `not in ['None', None]`